README.md

```diff
@@ -15,51 +15,92 @@ When you are done:
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
-# copy & paste your solution here
+def return_max(l):
+  if len(l) < 1:
+    return 'None'
+  else:
+    return(max(l))
 ```
-
 - [Min Value](https://repl.it/@Admin7/minvalue)
+
 ```python
-# copy & paste your solution here
+
+def return_max(l):
+  if len(l) < 1:
+    return 'None'
+  else:
+    return(min(l))
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
-# copy & paste your solution here
+def return_extended(l, iter):
+  for i in iter:
+    l.append(i)
+  return l
 ```
 
 - [Extend](https://repl.it/@Admin7/creatematrix)
 ```python
-# copy & paste your solution here
+def create_matrix(h, w):
+  result = []
+  for i in range(h):
+    row = []
+    for x in range(w):
+      row.append('#')
+    result.append(row)
+  return result
 ```
 
 
 #### Dictionaries
 - [Receipt Tally](https://repl.it/@Admin7/receipttally)
 ```python
-# copy & paste your solution here
+def get_total(h, w):
+  total = 0
+  new = 0
+  for i in h:
+    for key,value in i.items():
+        if value == w:
+            total = i.get('subtotal')
+  return total
 ```
 
 - [Translation](https://repl.it/@Admin7/translations)
 ```python
-# copy & paste your solution here
+def translation(word, lang):
+  ans = ''
+  for key,value in translations.items():
+    if key == word:
+      ans = translations[key]
+      ans = ans.get(lang)
+  return ans
 ```
 
 - [Check RSVP](https://repl.it/@Admin7/checkrsvp)
 ```python
-# copy & paste your solution here
+def check_rsvp(event):
+  attending = []
+  for key, value in event.items():
+    if value == True:
+      attending.append(value)
+  return len(attending)
```

Comment:
This works! But as we become more advanced Python programmers and write more involved applications, it's worth noting that this list will use up unnecessary space: n for every person attending. Instead, we can just keep a count of people who said they were attending. A number would have constant space complexity!